test/unit/spec/nbextensions/bulkImportCell/tabs/xsvGenerator-spec.js

```diff
@@ -255,58 +318,12 @@ define([
             });
 
-            describe('createForm', () => {
-                function checkFormInputs(xsvGen, form, isMulti = false) {
-                    const formParams = ['output_directory', 'output_file_type', 'types'];
-                    formParams.forEach((param) => {
-                        const paramElement = form.querySelector(`[name="${param}"]`);
-                        expect(paramElement).toBeDefined();
-                        if (param === 'output_file_type') {
-                            // ensure that all the output file types are present
-                            ['CSV', 'TSV', 'EXCEL'].forEach((type) => {
-                                expect(paramElement.innerHTML).toMatch(`value="${type}"`);
-                            });
-                        } else if (param === 'types') {
-                            const modelParams = xsvGen.model.getItem('params');
-                            // ensure that all the input types are present
-                            Object.keys(modelParams).forEach((type) => {
-                                expect(paramElement.innerHTML).toMatch(`value="${type}"`);
-                            });
-                        }
-                    });
-                    const multiFileInfo =
-                        'Choosing Excel as output file type will create a single file with a page for each';
-                    const guidanceSection = form.querySelector(
-                        `.${cssBaseClass}__paragraph--guide`
-                    );
-                    if (isMulti) {
-                        expect(guidanceSection.textContent).toContain(multiFileInfo);
-                    } else {
-                        expect(guidanceSection.textContent).not.toContain(multiFileInfo);
-                    }
-                }
-
-                it('creates a form with the appropriate inputs, single type', function () {
-                    this.xsvGen = new XSVGenerator({
-                        model: singleModel,
-                        typesToFiles,
-                        fileTypeMapping,
-                    });
-                    const form = document.createElement('div');
-                    form.innerHTML = this.xsvGen.createForm();
-                    checkFormInputs(this.xsvGen, form);
-                });
-
-                it('creates a form with the appropriate inputs, multiple types', function () {
-                    this.xsvGen = createXsvGen();
-                    const form = document.createElement('div');
-                    form.innerHTML = this.xsvGen.createForm();
-                    checkFormInputs(this.xsvGen, form, true);
-                });
+            xdescribe('createForm', () => {
+                // TODO!
             });
 
             describe('renderLayout', () => {
-                function checkStartUpDialog(modelData, hasError) {
+                function checkStartUpDialog(modelData, hasError, isMulti = false) {
```

Comment:
I combined the checks that had been in the old `createForm` tests into these `renderLayout` tests for simplicity